docs/scenarios/stacks.feature

```diff
@@ -1,99 +1,62 @@
-
-    Given that I am on a work page that has multiple items
-    When I have selected a number of those items to request
-    And have clicked a ‘Request It’ button
-    Then I see a pop up modal that shows an editable list of the items I’m requesting
-    And how many requests I have left
-    Then I can click a button to confirm my request
-    And see a ‘request confirmed’ message
-
-
-Feature: As a library member of staff that is logged in, I want to place a request at more than one time
-
-  Scenario: A logged in user places a request for different items at several times during a day/week
-
-    Given that I have requested an item previously
-    When I want to select more items to request
-    Then I want to see what items I’ve requested so far
-    And how many more I can request before I hit the limit
-
-
-Feature: As a library member of staff that is logged in, I want to place a request for multiple items at more than one time
-
-  Scenario: A logged in user places a request for multiple items at several times during a day/week
-
-    Given that I have already requested items previously
-    When I want to select more items to request at a later time
-    Then I need to see if an item has already been requested
-    And I need to see what I’ve requested so far
-    And how many more I can request before I hit the limit
+Feature: 1. As a library member I want to know if an item is available for requesting
+
+  Scenario: A user viewing a work page
+    Given the work has a requestable item 
+    And it’s available to request
+    Then I can see a that it's requestable
+  
+  Scenario: A user viewing a work page
+    Given the work has a requestable item 
+    And it’s unavailable to request
+    Then I can see a that it's temporarily unavailable
+  
+  Scenario: A user viewing a work page
+    Given the work has no requestable items
+    Then I can see a that it's not requestable and why (closed or open shelves or unknown)
+
+Feature: 2. As a library member I want to request an item
+
+  Scenario: A logged out user viewing a work page
+    Given the work has a requestable item 
+    And it’s available to request
+    Then I'm can see a primary CTA to log in
+    
+  Scenario: A logged out user viewing a work page
+    Given the work has a requestable item 
+    And it’s unavailable to request
+    Then I can't see a primary CTA to log in
+    
+  Scenario: A logged out user viewing a work page
+    Given the work has no requestable items
+    Then I can't see a primary CTA to log in
+    
+  Scenario: A logged in user viewing a work page
+    Given the work has a requestable item 
+    And it’s available to request
+    Then I can see a primary CTA to request it
+    
+  Scenario: A work has a single requestable item
+    Given that I am on a work page that has a single requestable item
+    When a user clicks the primary CTA to request the item
+    Then I receive feedback as to whether the request succeeded or failed
```

Comment:
Wondering (but don't know) if this could/should be split into multiple scenarios:

1. checks you can request the item
2. (assumes you've already requested as part of the scenario) checks you get location info if successful 
3. (assumes you've already requested as part of the scenario) checks you get some other message if unsuccessful

?